src/main/java/seedu/address/model/OfficeConnectModel.java

```diff
@@ -38,4 +41,27 @@ public RepositoryModelManager<AssignTask> getAssignTaskModelManager() {
         return assignTaskModelManager;
     }
 
+    public boolean hasItem(Task task) {
+        return taskModelManager.hasItem(task);
+    }
+
+    public void addItem(Task task) {
+        taskModelManager.addItem(task);
+    }
+
+    public void updateFilteredItemList(Predicate<Task> predicate) {
+        taskModelManager.updateFilteredItemList(predicate);
+    }
+
+    public ObservableList<Task> getFilteredItemList() {
+        return taskModelManager.getFilteredItemList();
+    }
+
+    public void deleteItem(Task task) {
+        taskModelManager.deleteItem(task);
+    }
+
+    public ReadOnlyRepository<Task> getReadOnlyRepository() {
+        return taskModelManager.getReadOnlyRepository();
+    }
```

Comment:
Name for this functions need to be specific since OfficeConnectModel contain multiple managers

Reply:
oops ok great catch!